src/layout.js

```diff
@@ -127,50 +128,63 @@ const makeGraph = function( nodes, edges, options ){
 
       kp.edges.push( k );
     } else {*/
-    graph.edges.push( k );
+    graph.edges.push(k);
     //}
   }
 
   return graph;
 };
 
-function Layout( options ){
-  const elkOptions = options.elk;
-  const cy = options.cy;
-
-  this.options = assign( {}, defaults, options );
-
-  this.options.elk = assign( {
-    aspectRatio: cy.width() / cy.height()
-  }, defaults.elk, elkOptions, elkOverrides );
-}
-
-Layout.prototype.run = function() {
-  const layout = this;
-  const options = this.options;
-
-  const eles = options.eles;
-  const nodes = eles.nodes();
-  const edges = eles.edges();
-
-  const elk = new ELK();
-  const graph = makeGraph( nodes, edges, options );
-
-  elk.layout(graph, {
-    layoutOptions: options.elk
-  }).then(() => {
-    nodes.filter(n => !n.isParent()).layoutPositions( layout, options, n => getPos(n, options) );
-  });
+class Layout {
```

Comment:
Using a `class` won't work in older browsers unless webpack/babel compiles this down to a function.  Is that the case?

I remember webpack did some weird stuff with compiling `class`es before.  Have you tested that the layout works in IE9 with your changes?

Reply:
Nothing in the readme even implies IE9 is a target, so I didn't know that'd be an issue.  
According to https://www.caniuse.com/usage-table IE9's global share is 0.21% at the time of writing, anyway.

(Have you tested `master` works in IE9, for that matter?)

That said, sure, we can add a `browserslist` to have Babel target ie9 too? Something like https://github.com/cytoscape/cytoscape.js/blob/unstable/.browserslist ?

Reply:
See https://js.cytoscape.org/#introduction/factsheet

Reply:
I added the browserslist from Cytoscape.js proper in f9b805231bd50356b7766f69c696103d3b95263e. Classes are now compiled down to plain old functions.